src/azure-cli/azure/cli/command_modules/synapse/manual/operations/sparkpool.py

```diff
@@ -96,11 +96,15 @@ def update_spark_pool(cmd, client, resource_group_name, workspace_name, spark_po
 
     if package_action and package:
         if package_action == "Add":
+            import datetime
             if existing_spark_pool.custom_libraries is None:
                 existing_spark_pool.custom_libraries = []
             for item in package:
                 package_get = get_workspace_package(cmd, workspace_name, item)
-                library = LibraryInfo(name=package_get.name, type=package_get.properties.type, path=package_get.properties.path, container_name=package_get.properties.container_name)
+                uploadtime = datetime.datetime.now()
```

Comment:
Better to use `now(timezone.utc)` instead of local time as the same code may run in different time zone?